packages/xstate-wallet/src/serde/app-messages/serialize.ts

```diff
@@ -1,11 +1,35 @@
 import {
   Allocation as AppAllocation,
   Allocations as AppAllocations,
-  AllocationItem as AppAllocationItem
+  AllocationItem as AppAllocationItem,
+  SiteBudget as AppSiteBudget
 } from '@statechannels/client-api-schema';
-import {Allocation, AllocationItem, SimpleAllocation} from '../../store/types';
-import {tokenAddress} from '../../constants';
+import {
+  Allocation,
+  AllocationItem,
+  SimpleAllocation,
+  SiteBudget,
+  BudgetItem
+} from '../../store/types';
+import {tokenAddress, ETH_ASSET_HOLDER_ADDRESS} from '../../constants';
 
+export function serializeSiteBudget(budget: SiteBudget): AppSiteBudget {
+  const assetBudget = budget.budgets[ETH_ASSET_HOLDER_ADDRESS];
```

Comment:
Can we avoid hard-coding ETH inside the wallet if it isn't too much extra to add an `assetType` or `tokenAddress` field to the budget object?

Reply:
Sure, I've added support for multiple assets.